<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusinessMenu</class>
 <widget class="QDialog" name="BusinessMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New Entry</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_session">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name_session">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_room_session">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time_session">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_session">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete_session">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>720</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Conference Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Conference Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Session Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Room Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>630</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Session Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sessions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferences</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_equipmentUsed">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>50</y>
     <width>201</width>
     <height>741</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipment Used</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipment Available</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_equipmentAvailable">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>50</y>
     <width>201</width>
     <height>741</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
